src/adv.py

```diff
@@ -43,20 +43,32 @@
 
 print(player)
 # Write a loop that:
-#
+
+def move(direction, current_room):
+    attribute = direction + '_to'
+
+    if hasattr(current_room, attribute):
+        return getattr(current_room, attribute)
+    else:
+        print("Sorry, the path is blocked, find another way!! \n")
+        return current_room
+
+
+while True:
 # * Prints the current room name
+    print("You are currently in the", player.current_room.name)
 # * Prints the current description (the textwrap module might be useful here).
+    print("Take note,", player.current_room.description)
 # * Waits for user input and decides what to do.
-#
+    cmd = input("\n> please input here: ").lower()
 # If the user enters a cardinal direction, attempt to move to the room there.
+
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
-
-while True:
-    cmd = input("please input here: ")
```

Comment:
You did a really good job. One suggestion I have is to handle inputs that are more than just one letter. you will need this not only if you are going to be picking up or dropping objects, but also if you have a new user coming to the game who doesn't know the instructions. 

The way I went about doing this was to have a check to see if the input or arguments in the command line were greater than 1. if it was just 1, then I would make that input lowercase and only use the first letter so I could move. But if it was more than 1 argument, I would use the full sentence. example: drop item. grab rock. etc.